src/escalus_stanza.erl

```diff
@@ -893,15 +893,20 @@ xml_to_xmlterm(XML) when is_binary(XML) ->
       Snippet :: xml_snippet(),
       Ctx :: context(),
       Text :: string().
+
 render(Snippet, Ctx) ->
-    mustache:render(xml_snippet_to_string(Snippet),
-                    validate_context(Ctx)).
+    ValidateCtx = validate_context(Ctx),
+    ct:pal("WTF: ~p~n~p", [Ctx, ValidateCtx]),
+    R = bbmustache:render(xml_snippet_to_binary(Snippet),
+                      ValidateCtx),
+    ct:pal("Render: ~p~n~p", [Snippet, R]),
```

Comment:
Call to `ct` outside `escalus_ct`

Reply:
@erszcz I just removed and the other call to `ct:pal`.